test/openai/stream_test.exs

```diff
@@ -39,7 +48,6 @@ defmodule OpenAI.StreamTest do
         |> Enum.to_list()
 
       assert [
-               %{"choices" => [], "code" => 400, "status" => :error},
```

Comment:
Just a heads up I changed the error format slightly.  I'm not sure if we should keep that contract for the user but considering this is a pre rel 1 release I pulled this out since it seemed like a bit of a work-around given the way the stream worked in the previous versions.